Katello/katello
# Fixes #34676 - Incremental CV update does not auto-publish CCV

#### What are the changes introduced in this pull request?
"Auto publish" was not considered during incremental update.  This is fixed by adding a check for `auto_publish?` where components are added for `propagate_all_composites`.

#### Considerations taken when implementing this change?
None really, it's straight forward.

#### What are the testing steps for this pull request?
1) Incremental update a component CV that's within a composite that has auto publish on
  -> Ensure that the composite gets a new point-release version
2) Try the same thing using propagate-all-composites
3) Ensure that inc update still works fine in the UI
